site/src/routes/docs/_sections.js

```diff
@@ -85,6 +86,10 @@ export default function() {
 			renderer.heading = (text, level, rawtext) => {
 				let slug;
 
+				let extracted = extract_attributes(text, rawtext);
```

Comment:
I think this could be `const` instead of `let`

Reply:
Fixed in latest commit.